autoload/neomake/signs.vim

```diff
@@ -125,8 +125,15 @@ function! neomake#signs#PlaceVisibleSigns() abort
         if !has_key(s:sign_queue[type], buf)
             continue
         endif
-        let topline = line('w0')
-        let botline = line('w$')
+
+        if !get(g:, 'neomake_place_signs_at_once', 0)
+            let topline = line('w0')
+            let botline = line('w$')
+        else
+            let topline = line('0')
+            let botline = line('$')
```

Comment:
What about switching the blocks, and having the true-case first (removing the `!` there).
Seems to be more readable?!